run_thread_switcher_aida64.bat

```diff
@@ -0,0 +1,7 @@
+pushd %~dp0
+
+cp settings_aida64.py settings.py
+
+python310 main.py
```

Comment:
How about setting the python command as `py` by default? I think most people would download the interpreter from python.org, and thus it would be a good default. What do you think?

Reply:
```suggestion
py main.py
```